packages/helm-charts/testnet/templates/_helpers.tpl

```diff
@@ -1,390 +0,0 @@
-{{/* vim: set filetype=mustache: */}}
-{{/*
-Expand the name of the chart.
-*/}}
-{{- define "ethereum.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
-
-{{/*
-Create a default fully qualified app name.
-We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
-If release name contains chart name it will be used as a full name.
-*/}}
-{{- define "ethereum.fullname" -}}
-{{- if .Values.fullnameOverride -}}
-{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
-{{- else -}}
-{{- $name := "" -}}
-{{- if contains $name .Release.Name -}}
-{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
-{{- else -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
-{{- end -}}
-{{- end -}}
-
-{{/*
-Create chart name and version as used by the chart label.
-*/}}
-{{- define "ethereum.chart" -}}
-{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
-
-{{- define "standard.labels" -}}
-{{- include "standard.short_labels" . }}
-chart: {{ template "ethereum.chart" . }}
-heritage: {{ .Release.Service }}
-{{- end -}}
-
-{{- define "standard.short_labels" -}}
-app: {{ template "ethereum.name" . }}
-release: {{ .Release.Name }}
-{{- end -}}
-
-
-{{- define "celo.init-genesis-container" -}}
```

Comment:
Although a lot of this is dead code, we want to keep some of these (like this is still used by the validator statefulset). I deleted the dead parts of this file in the proxy infrastructure PR that's hopefully going to be merged soon, so if you want you can leave this file untouched and I can clean it up with that PR.

Reply:
I've reinstated this file minus setting the nodekey for the miner.